src/app/shared/issue-tables/IssuesDataTable.ts

```diff
@@ -159,6 +161,17 @@ export class IssuesDataTable extends DataSource<Issue> {
     return result;
   }
 
+  private todoFinished(issue: Issue): number {
+    let count = 0;
+    for (const dispute of issue.issueDisputes) {
+      if (dispute.isDone()) {
+        count += 1;
+      }
+    }
+
+    return count;
```

Comment:
Can use a reducer to reduce the array into the result you want instead of a for loop here.